ember-server/shared/src/main/scala/org/http4s/ember/server/internal/ServerHelpers.scala

```diff
@@ -359,7 +359,7 @@ private[server] object ServerHelpers extends ServerHelpersPlatform {
       .drain
       .attempt
       .flatMap {
-        case Left(err) => onWriteFailure(request, resp, err)
+        case Left(err) => onWriteFailure(request, resp, err) >> MonadThrow[F].raiseError(err)
```

Comment:
A teeny-weeny nitpick
```suggestion
        case Left(err) => onWriteFailure(request, resp, err) *> MonadThrow[F].raiseError(err)
```